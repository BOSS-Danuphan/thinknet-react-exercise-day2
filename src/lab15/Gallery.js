import React, { Component } from 'react'
import styled from 'styled-components'
export const StyledH1 = styled.h1`
color: red;
font-size: 10px;
`
class Gallery extends Component {
  render() {
    return <StyledH1>Gallery</StyledH1>
  }
}
// Check package rewire
export default Gallery