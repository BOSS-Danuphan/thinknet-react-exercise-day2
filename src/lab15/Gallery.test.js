import React from 'react'
import { shallow } from 'enzyme'
import Gallery, {StyledH1} from './Gallery'

it('Snapshot testing', () => {
  const wrapper = shallow(<Gallery />, { disableLifecycleMethods: true });
  expect(wrapper).toMatchSnapshot()
});

it('Snapshot testing', () => {
  const wrapper = shallow(<StyledH1 />);
  expect(wrapper).toMatchSnapshot()
});
