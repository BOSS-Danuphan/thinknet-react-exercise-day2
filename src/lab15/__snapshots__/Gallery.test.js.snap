// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing 1`] = `
<styled.h1>
  Gallery
</styled.h1>
`;

exports[`Snapshot testing 2`] = `
.c0 {
  color: red;
  font-size: 10px;
}

<h1
  className="c0"
/>
`;
